algorithms/Python/searching/Uniform_Cost_Search .py

```diff
@@ -0,0 +1,44 @@
+# this graph to check the algorithm
+graph={
+  'S':[('A',2),('B',3),('D',5)],
+  'A':[('C',4)],
+  'B':[('D',4)],
+  'C':[('D',1),('G',2)],
+  'D':[('G',5)],
+  'G':[],
+    }
+
+#to calculate the total cost of path
+def path_cost(path):
+    total_cost=0
+    for (node, cost) in path:
+        total_cost+=cost
+    return total_cost , path[-1][0]
+# well sort queue items based on total path
+#if two items have the same total_cost then sort by node name (alphabeticaly)

```

Comment:
Please change the  (alphabeticaly) ->to  (alphabetically ) here. It failed because of the spell mistake in the comment. your code is good. please change the spelling.

Reply:
I make it, thank sir